LemonTart.html

```diff
@@ -0,0 +1,75 @@
+<!DOCTYPE html>
+<html>
+  <head>
+    <title>Lemon Tart Recipe</title>
+    <style>
+      img.Lemon{
+        height: 50%;
+        width: 50%
+      }
+    </style>
```

Comment:
hmmmm we did not go over styling, and this seems a bit off to me as well. If it works, this is definitely not standard practice :D I would rather recommend something like 
```suggestion
    <style>
      .Lemon {
        height: 50%;
        width: 50%
      }
    </style>
```